myproject/leave/tests.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+from .models import User, LeaveRequest
+from django.core.urlresolvers import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase, APIRequestFactory
+from .serializers import LeaveRequestSerializer
+# Create your tests here.
+# factory = APIRequestFactory()
+# request = factory.get('/approval/OmPrakash',{'name':'OmPrakash'},format='json')
+
+class LeaveRequestTest(APITestCase):
+	def setUp(self):
+		self.client = Client()
+		self.username = 'admin'
+		self.email = 'admin@gmail.com'
+		self.password = 'test1234'
+		self.test_user = User.objects.create_user(self.username, self.email, self.password)
+		login = self.client.login(username=self.username, password=self.password)
+		self.assertEqual(login, True)
```

Comment:
set up method is used to create dependencies for upcoming tests. No need for assertions required here.